src/raftstore/store/bootstrap.rs

```diff
@@ -65,7 +65,9 @@ pub fn bootstrap_store(engines: &Engines, cluster_id: u64, store_id: u64) -> Res
     ident.set_cluster_id(cluster_id);
     ident.set_store_id(store_id);
 
-    engines.kv_engine.put_msg(&ident_key, &ident)
+    try!(engines.kv_engine.put_msg(&ident_key, &ident));
+    try!(engines.kv_engine.flush_wal(true));
```

Comment:
why not use sync_wal?

Reply:
sync_wal wouln't call flush_wal, so wal buffer may not flush